crates/oxc_prettier/src/lib.rs

```diff
@@ -37,6 +38,12 @@ impl GroupIdBuilder {
     }
 }
 
+#[derive(Debug, Default)]
+pub struct PrettierArgs {
+    expand_first_arg: bool,
+    expand_last_arg: bool,
+}
+
```

Comment:
In support of https://github.com/prettier/prettier/blob/main/src/language-js/print/arrow-function.js#L171-L172